.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -118,6 +161,66 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.destination-slot-name }}
 
+      - name: Checkout Code Repository
```

Comment:
This should do a checkout without submodules.

Reply:
Our current checkout action doesn't have a passthrough for this hand hardcodes it to "recursive" would you like be to change the checkout action to accept this optional parameter?

Reply:
Yeah sure, just copy the action's original parameter there so that's also proxied.